src/node/http.ts

```diff
@@ -62,6 +62,10 @@ export const replaceTemplates = <T extends object>(
     .replace("{{OPTIONS}}", () => escapeJSON(serverOptions))
 }
 
+export enum Cookie {
+  SessionKey = "code-server-session",
```

Comment:
Renamed from ‘key’ in the browser to assist in debugging, both from us and users.